XenModel/Actions/CrossConnectionParallelAction.cs

```diff
@@ -0,0 +1,158 @@
+﻿/* Copyright (c) Citrix Systems Inc. 
```

Comment:
We should consider whether CCPA should be cancellable (cancelling those sub-actions that we can, and not launching any new ones). This might be useful later.